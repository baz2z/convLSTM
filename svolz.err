usage: test.py [-h] [--model_idx MODEL_IDX]
test.py: error: unrecognized arguments: --run_idx 0
usage: test.py [-h] [--model_idx MODEL_IDX]
test.py: error: unrecognized arguments: --run_idx 1
