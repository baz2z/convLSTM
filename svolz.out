/mnt/qb/work/butz/svolz67/dev/models
JOB DATA
JobId=1420180 ArrayJobId=1420180 ArrayTaskId=3 JobName=review.sh
   UserId=svolz67(4458) GroupId=butz(4032) MCS_label=N/A
   Priority=38833 Nice=0 Account=butz QOS=normal
   JobState=RUNNING Reason=None Dependency=(null)
   Requeue=1 Restarts=0 BatchFlag=1 Reboot=0 ExitCode=0:0
   RunTime=00:00:02 TimeLimit=00:30:00 TimeMin=N/A
   SubmitTime=2022-05-12T14:29:11 EligibleTime=2022-05-12T14:29:11
   AccrueTime=Unknown
   StartTime=2022-05-12T14:29:11 EndTime=2022-05-12T14:59:11 Deadline=N/A
   PreemptEligibleTime=2022-05-12T14:30:11 PreemptTime=None
   SuspendTime=None SecsPreSuspend=0 LastSchedEval=2022-05-12T14:29:11
   Partition=gpu-2080ti AllocNode:Sid=bg-slurmb-login1:9756
   ReqNodeList=(null) ExcNodeList=(null)
   NodeList=slurm-bm-30
   BatchHost=slurm-bm-30
   NumNodes=1 NumCPUs=8 NumTasks=1 CPUs/Task=1 ReqB:S:C:T=0:0:*:*
   TRES=cpu=8,mem=3G,node=1,billing=2,gres/gpu=1
   Socks/Node=* NtasksPerN:B:S:C=0:0:*:* CoreSpec=*
   MinCPUsNode=1 MinMemoryNode=3G MinTmpDiskNode=0
   Features=(null) DelayBoot=00:00:00
   OverSubscribe=OK Contiguous=0 Licenses=(null) Network=(null)
   Command=/mnt/qb/work/butz/svolz67/dev/models/review.sh
   WorkDir=/mnt/qb/work/butz/svolz67/dev/models
   StdErr=/mnt/qb/work/butz/svolz67/dev/models/svolz.err
   StdIn=/dev/null
   StdOut=/mnt/qb/work/butz/svolz67/dev/models/svolz.out
   Power=
   CpusPerTres=gpu:8
   TresPerNode=gpu:1
   MailUser=sebastien.volz@student.uni-tuebingen.de MailType=FAIL
   NtasksPerTRES:0

JobId=1420183 ArrayJobId=1420180 ArrayTaskId=2 JobName=review.sh
   UserId=svolz67(4458) GroupId=butz(4032) MCS_label=N/A
   Priority=38833 Nice=0 Account=butz QOS=normal
   JobState=RUNNING Reason=None Dependency=(null)
   Requeue=1 Restarts=0 BatchFlag=1 Reboot=0 ExitCode=0:0
   RunTime=00:00:02 TimeLimit=00:30:00 TimeMin=N/A
   SubmitTime=2022-05-12T14:29:11 EligibleTime=2022-05-12T14:29:11
   AccrueTime=Unknown
   StartTime=2022-05-12T14:29:11 EndTime=2022-05-12T14:59:11 Deadline=N/A
   PreemptEligibleTime=2022-05-12T14:30:11 PreemptTime=None
   SuspendTime=None SecsPreSuspend=0 LastSchedEval=2022-05-12T14:29:11
   Partition=gpu-2080ti AllocNode:Sid=bg-slurmb-login1:9756
   ReqNodeList=(null) ExcNodeList=(null)
   NodeList=slurm-bm-30
   BatchHost=slurm-bm-30
   NumNodes=1 NumCPUs=8 NumTasks=1 CPUs/Task=1 ReqB:S:C:T=0:0:*:*
   TRES=cpu=8,mem=3G,node=1,billing=2,gres/gpu=1
   Socks/Node=* NtasksPerN:B:S:C=0:0:*:* CoreSpec=*
   MinCPUsNode=1 MinMemoryNode=3G MinTmpDiskNode=0
   Features=(null) DelayBoot=00:00:00
   OverSubscribe=OK Contiguous=0 Licenses=(null) Network=(null)
   Command=/mnt/qb/work/butz/svolz67/dev/models/review.sh
   WorkDir=/mnt/qb/work/butz/svolz67/dev/models
   StdErr=/mnt/qb/work/butz/svolz67/dev/models/svolz.err
   StdIn=/dev/null
   StdOut=/mnt/qb/work/butz/svolz67/dev/models/svolz.out
   Power=
   CpusPerTres=gpu:8
   TresPerNode=gpu:1
   MailUser=sebastien.volz@student.uni-tuebingen.de MailType=FAIL
   NtasksPerTRES:0

JobId=1420182 ArrayJobId=1420180 ArrayTaskId=1 JobName=review.sh
   UserId=svolz67(4458) GroupId=butz(4032) MCS_label=N/A
   Priority=38833 Nice=0 Account=butz QOS=normal
   JobState=RUNNING Reason=None Dependency=(null)
   Requeue=1 Restarts=0 BatchFlag=1 Reboot=0 ExitCode=0:0
   RunTime=00:00:02 TimeLimit=00:30:00 TimeMin=N/A
   SubmitTime=2022-05-12T14:29:11 EligibleTime=2022-05-12T14:29:11
   AccrueTime=Unknown
   StartTime=2022-05-12T14:29:11 EndTime=2022-05-12T14:59:11 Deadline=N/A
   PreemptEligibleTime=2022-05-12T14:30:11 PreemptTime=None
   SuspendTime=None SecsPreSuspend=0 LastSchedEval=2022-05-12T14:29:11
   Partition=gpu-2080ti AllocNode:Sid=bg-slurmb-login1:9756
   ReqNodeList=(null) ExcNodeList=(null)
   NodeList=slurm-bm-27
   BatchHost=slurm-bm-27
   NumNodes=1 NumCPUs=8 NumTasks=1 CPUs/Task=1 ReqB:S:C:T=0:0:*:*
   TRES=cpu=8,mem=3G,node=1,billing=2,gres/gpu=1
   Socks/Node=* NtasksPerN:B:S:C=0:0:*:* CoreSpec=*
   MinCPUsNode=1 MinMemoryNode=3G MinTmpDiskNode=0
   Features=(null) DelayBoot=00:00:00
   OverSubscribe=OK Contiguous=0 Licenses=(null) Network=(null)
   Command=/mnt/qb/work/butz/svolz67/dev/models/review.sh
   WorkDir=/mnt/qb/work/butz/svolz67/dev/models
   StdErr=/mnt/qb/work/butz/svolz67/dev/models/svolz.err
   StdIn=/dev/null
   StdOut=/mnt/qb/work/butz/svolz67/dev/models/svolz.out
   Power=
   CpusPerTres=gpu:8
   TresPerNode=gpu:1
   MailUser=sebastien.volz@student.uni-tuebingen.de MailType=FAIL
   NtasksPerTRES:0

JobId=1420181 ArrayJobId=1420180 ArrayTaskId=0 JobName=review.sh
   UserId=svolz67(4458) GroupId=butz(4032) MCS_label=N/A
   Priority=38833 Nice=0 Account=butz QOS=normal
   JobState=RUNNING Reason=None Dependency=(null)
   Requeue=1 Restarts=0 BatchFlag=1 Reboot=0 ExitCode=0:0
   RunTime=00:00:02 TimeLimit=00:30:00 TimeMin=N/A
   SubmitTime=2022-05-12T14:29:11 EligibleTime=2022-05-12T14:29:11
   AccrueTime=Unknown
   StartTime=2022-05-12T14:29:11 EndTime=2022-05-12T14:59:11 Deadline=N/A
   PreemptEligibleTime=2022-05-12T14:30:11 PreemptTime=None
   SuspendTime=None SecsPreSuspend=0 LastSchedEval=2022-05-12T14:29:11
   Partition=gpu-2080ti AllocNode:Sid=bg-slurmb-login1:9756
   ReqNodeList=(null) ExcNodeList=(null)
   NodeList=slurm-bm-27
   BatchHost=slurm-bm-27
   NumNodes=1 NumCPUs=8 NumTasks=1 CPUs/Task=1 ReqB:S:C:T=0:0:*:*
   TRES=cpu=8,mem=3G,node=1,billing=2,gres/gpu=1
   Socks/Node=* NtasksPerN:B:S:C=0:0:*:* CoreSpec=*
   MinCPUsNode=1 MinMemoryNode=3G MinTmpDiskNode=0
   Features=(null) DelayBoot=00:00:00
   OverSubscribe=OK Contiguous=0 Licenses=(null) Network=(null)
   Command=/mnt/qb/work/butz/svolz67/dev/models/review.sh
   WorkDir=/mnt/qb/work/butz/svolz67/dev/models
   StdErr=/mnt/qb/work/butz/svolz67/dev/models/svolz.err
   StdIn=/dev/null
   StdOut=/mnt/qb/work/butz/svolz67/dev/models/svolz.out
   Power=
   CpusPerTres=gpu:8
   TresPerNode=gpu:1
   MailUser=sebastien.volz@student.uni-tuebingen.de MailType=FAIL
   NtasksPerTRES:0

node:
slurm-bm-30.novalocal
slurm-bm-27.novalocal
RUN Script
