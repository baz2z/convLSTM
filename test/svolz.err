Traceback (most recent call last):
  File "/mnt/qb/work/butz/svolz67/dev/convLSTM/test/./testLR.py", line 263, in <module>
    loss = calcLoss(model, context, horizon, dataloader)
  File "/mnt/qb/work/butz/svolz67/dev/convLSTM/test/./testLR.py", line 209, in calcLoss
    os.chdir(f'./run{runNbr}')
FileNotFoundError: [Errno 2] No such file or directory: './run1'
