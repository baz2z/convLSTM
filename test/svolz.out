/mnt/qb/work/butz/svolz67/dev/convLSTM/test
JOB DATA
JobId=1630360 ArrayJobId=1630360 ArrayTaskId=1 JobName=review.sh
   UserId=svolz67(4458) GroupId=butz(4032) MCS_label=N/A
   Priority=19852 Nice=0 Account=butz QOS=normal
   JobState=RUNNING Reason=None Dependency=(null)
   Requeue=1 Restarts=0 BatchFlag=1 Reboot=0 ExitCode=0:0
   RunTime=00:00:00 TimeLimit=3-00:00:00 TimeMin=N/A
   SubmitTime=2022-07-17T18:41:52 EligibleTime=2022-07-17T18:41:52
   AccrueTime=2022-07-17T18:42:03
   StartTime=2022-07-17T18:42:25 EndTime=2022-07-20T18:42:25 Deadline=N/A
   PreemptEligibleTime=2022-07-17T18:43:25 PreemptTime=None
   SuspendTime=None SecsPreSuspend=0 LastSchedEval=2022-07-17T18:42:25
   Partition=gpu-2080ti AllocNode:Sid=bg-slurmb-login1:32564
   ReqNodeList=(null) ExcNodeList=(null)
   NodeList=slurm-bm-83
   BatchHost=slurm-bm-83
   NumNodes=1 NumCPUs=8 NumTasks=1 CPUs/Task=1 ReqB:S:C:T=0:0:*:*
   TRES=cpu=8,mem=3G,node=1,billing=2,gres/gpu=1
   Socks/Node=* NtasksPerN:B:S:C=0:0:*:* CoreSpec=*
   MinCPUsNode=1 MinMemoryNode=3G MinTmpDiskNode=0
   Features=(null) DelayBoot=00:00:00
   OverSubscribe=OK Contiguous=0 Licenses=(null) Network=(null)
   Command=/mnt/qb/work/butz/svolz67/dev/convLSTM/test/review.sh
   WorkDir=/mnt/qb/work/butz/svolz67/dev/convLSTM/test
   StdErr=/mnt/qb/work/butz/svolz67/dev/convLSTM/test/svolz.err
   StdIn=/dev/null
   StdOut=/mnt/qb/work/butz/svolz67/dev/convLSTM/test/svolz.out
   Power=
   CpusPerTres=gpu:8
   TresPerNode=gpu:1
   MailUser=sebastien.volz@student.uni-tuebingen.de MailType=FAIL
   NtasksPerTRES:0

node:
slurm-bm-83.novalocal
RUN Script
